Copy to SD Card root directory to update/config_rrf.ini

```diff
@@ -156,11 +156,19 @@ invert_axis:X0 Y0 Z0 LY0
 #   Options: [enable: 1, disable: 0]
 persistent_info:0
 
-#### File List Mode
+#### Files List Mode
 # Display files in list mode instead of icon mode.
 #   Options: [enable: 1, disable: 0]
 files_list_mode:1
 
+#### Files sorting
+# Sort files and folders based on the selected option.
+#
+# NOTE: Only applicable for files in TFT SD Card and TFT USB Stick.
+#
+#   Options: [Date Newest First: 0, Date Oldest First: 1, Name Ascending: 2, Name Descending: 3]
```

Comment:
shouldn't better someting like Newest, Oldest, Ascending, Descending? The param name and description are already self explaining

Reply:
How would you define what is ascending and descending?

Reply:
talking about standard file names ordering type by date and name. In any case, just my opinion